industrial_trajectory_filters/include/industrial_trajectory_filters/filter_base.h

```diff
@@ -37,7 +37,7 @@
 #include "ros/console.h"
 #include "ros/ros.h"
 #include <moveit/planning_request_adapter/planning_request_adapter.h>
-#include <class_loader/class_loader.h>
+#include <class_loader/class_loader.hpp>
```

Comment:
Are these changes backwards compatible with `kinetic` or does it require a new `melodic_devel` branch?

Reply:
The file changes are backward compatible to kinetic but the travis file directly references the new distro.  This builds successfully:
```
FROM osrf/ros:kinetic-desktop-full-xenial

RUN apt-get update && apt-get install -y \
    ros-kinetic-moveit \
    python-rosinstall \
    python-rosinstall-generator \
    python-wstool \
    build-essential \
    python-pip \
    nano \
    gdb \
    && rm -rf /var/lib/apt/lists/*

RUN pip install --upgrade pip

RUN pip install -U catkin_tools

RUN mkdir -p /root/ros_libs/src

WORKDIR /root/ros_libs/src

RUN git clone -b melodic-devel https://github.com/AustinDeric/industrial_core.git

WORKDIR /root/ros_libs/

RUN /bin/bash -c "source /opt/ros/kinetic/setup.bash && catkin build && catkin install"
```

I assumed you wanted to make a new branch for the corresponding bloom.

Reply:
latest commit passes travis as well: https://travis-ci.org/AustinDeric/industrial_core/jobs/413175715/config

Reply:
> The file changes are backward compatible to kinetic

if the changes are bw-compatible then branching would not seem to be necessary.

One less variant to maintain.

Reply:
Should we just add `melodic` instead of replace `kinetic`?

Reply:
Just trying to fix this error:
`E: Unable to locate package ros-melodic-industrial-core`

Reply:
>E: Unable to locate package ros-melodic-industrial-core

But Travis passes...maybe that's an outdated comment.

Reply:
>But Travis passes...maybe that's an outdated comment.
Correct, maybe a path forward is to just bloom release this kinetic-devel branch to the melodic distro?  That would work for me.